operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [CHANGE] Tokengen: Added k8s secret storage for the admin token. #5237
```

Comment:
can you clarify that now the token is stored in a secret by default?

Reply:
Your comment made me think about whether or not this should be enabled by default.  I decided to disable by default so chart consumers weren't surprised with new behavior.